docs/content/guides/technical-specification/third-party-licenses.md

```diff
@@ -56,26 +56,11 @@ The dependencies below apply only if you use the [`Formulas`](@/api/formulas.md)
     License: Apache v2.0<br>
     [https://github.com/SAP/chevrotain](https://github.com/SAP/chevrotain)
 
-- **core-js**<br>
-    Author: Denis Pushkarev<br>
-    License: Open source (MIT)<br>
-    [https://github.com/zloirock/core-js](https://github.com/zloirock/core-js)
```

Comment:
`core-js` is still our dependency. In the `package.json` of HyperFormula it is a `devDependency`, but snippets of code from `core-js` are bundled in the UMD build of HyperFormula.

We also [had a discussion](https://github.com/handsontable/hyperformula/pull/1030#discussion_r962926728) whether or not communicate it as a dependency in our docs and we decided to keep it.

Reply:
@sequba - got it, bringing it back: [880dd01](https://github.com/handsontable/handsontable/pull/9875/commits/880dd01c3e0432f6528f4d678950f3472a5f6447)